dist/unix/systemd/qbittorrent-nox@.service.in

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=qBittorrenti-nox service for user %I
+
+Documentation=man:qbittorrent-nox(1)
+After=network.target
+
+[Service]
+User=%i
+ExecStart=@EXPAND_BINDIR@/qbittorrent-nox
+Type=simple
+PrivateTmp=yes
```

Comment:
>But as you request, I'll remove it.

It's actually a bit more complex under the hood, if `PrivateTmp=true` you won't be able to add torrents to qbt via cmdline, I'm not sure users uses this extensively or not...
also: https://github.com/qbittorrent/qBittorrent/pull/6806#issuecomment-302861276